Make SNS documentation ready for publishing (#505)
dfinity/portal: docs/developer-docs/integrations/sns/index.md

# Service Nervous System (SNS)

We anticipate that different target groups are interested
in different aspects of the Service Nervous System (SNS)
and thus in different kinds of documentations.
We thus try to split the SNS documentation accordingly.

This page provides an overview of the SNS documentation and 
provides links to the parts that already exist.
Some of the SNS documentation is still being worked on and will be 
edited or added as the last SNS features are added, so please bare with
us if it is not yet complete or a few parts do not yet work.

## Technical documentation
This documentation is, as the name suggests, aimed at
developers.
Most of the documentation targets developers that have
a dapp that they would like to decentralize with an SNS. 
However, this documentation also targets developers that
want to build services that integrate with SNSs, such as wallet dapps
or decentralized exchanges.  

In this documentation you will find
* [A technical introduction to the SNS](sns-intro.md)

* Information on how to **get an SNS**, which includes
    * [An introduction how to get an SNS](get-sns/get-sns-intro.md) which

>       exlains the high level stages of gettin an SNS

Reviewer: ```suggestion
      explains the high level stages of getting an SNS
```

    * [Technical preparations for getting an SNS](get-sns/preparation.md) which

>       describes the technical steps required before getting an SNS. here
>       We also link to non-technical steps that we recommend,

Reviewer: ```suggestion
      describes the technical steps required before getting an SNS.
      We also link to non-technical steps that we recommend,
```

      which can be found under the "DAO and Tokenomics" documentation
      targeted at a less technical audience.
    * [Steps to Test the SNS locally](get-sns/local-testing.md)
    * Steps how to get an SNS in production <!--[Steps how to get an SNS in production](get-sns/get-sns-production.md)-->
  
* Tips for how to **integrate with an SNS**, which includes
  * Guidelines how to integrate a frontend <!--[Guidelines how to integrate a frontend](integrate-sns/frontend-integration.md)-->
  * [Guidelines how to integrate with the ledger canister](integrate-sns/ledger-integration.md)
  * [Guidelines how to integrate with the index canister](integrate-sns/index-integration.md)

* Guidelines for how to **manage an SNS**, which includes
  * [An introduction to managing an SNS](managing-sns/manage-sns-intro.md)
  * [Tips regarding cycles management for the cansiters](managing-sns/cycles-usage.md)
  * Information on nervous system parameters that can be configured
    in each SNS <!--[Information on nervous system parameters that can be configured
    in each SNS](managing-sns/nervous-system-parameters.md)-->

>   * Information no how SNS are upgraded <!--[Information no how SNS are upgraded](managing-sns/upgradeSNS.md)-->  

Reviewer: ```suggestion
  * Information on how SNS are upgraded <!--[Information on how SNS are upgraded](managing-sns/upgradeSNS.md)-->  
```

  * A guideline for SNS proposals <!--[A guideline for SNS proposals](managing-sns/proposal-guide.md)-->
 
## DAOs and Tokenomics documentation 
This documentation is less technical.
It targets for example developers who want to get an SNS and
have to make certain non-technical decisions too.

> It also targets interestd developers who would like to understand

Reviewer: ```suggestion
It also targets interested developers who would like to understand
```

the tokenomics of an SNS better.

In this documentation you will find

> * [An introduction to DAOs and
>   SNSs](../../../tokenomics/sns/sns-intro-tokens.md)
> * [An introduction to tokenomics of 
>   DAOs](../../../tokenomics/sns/tokenomics.md)
> * [An introduction to SNS 
>   rewards](../../../tokenomics/sns/rewards.md)
> * [Tips on what to consider before getting an
>   SNS](../../../tokenomics/sns/predeployment-considerations.md)

Reviewer: ```suggestion
* [An introduction to DAOs and SNSs](../../../tokenomics/sns/sns-intro-tokens.md)
* [An introduction to tokenomics of DAOs](../../../tokenomics/sns/tokenomics.md)
* [An introduction to SNS rewards](../../../tokenomics/sns/rewards.md)
* [Tips on what to consider before getting an SNS](../../../tokenomics/sns/predeployment-considerations.md)
```
linebreaks break links
